libqtile/widget/generic_poll_text.py

```diff
@@ -92,3 +92,47 @@ def poll(self):
             text = "Can't parse"
 
         return text
+
+    
+class GenPollCommand(base.ThreadPoolText):
+    """A generic text widget to display output from scripts or shell commands"""
+
+    defaults = [
+        ("update_interval", 60,"update time in seconds"),
+        ("cmd", "date", "command line as a string or list of arguments to execute"),
+        ("shell", True, "run command through a shell to enable piping and shell expansion"),
```

Comment:
```suggestion
        ("shell", False, "run command through a shell to enable piping and shell expansion"),
```
I'd be inclined to set this to `False`. Usage of `shell=True` is typically discouraged unless required (for the reasons you say).